packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/classes_table_single.dart

```diff
@@ -127,21 +127,15 @@ class ClassesTableSingle extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ValueListenableBuilder(
-      valueListenable: classes.filter,
-      builder: (_, __, ___) {
-        final data = classes.filtered();
-        final dataKey = 'ClassesTableSingle-$data';
-        return FlatTable<SingleClassStats>(
-          columns: _columns,
-          data: data,
-          dataKey: dataKey,
-          keyFactory: (e) => Key(dataKey),
-          selectionNotifier: selection,
-          defaultSortColumn: _shallowSizeColumn,
-          defaultSortDirection: SortDirection.descending,
-        );
-      },
+    final dataKey = 'ClassesTableSingle-${identityHashCode(classes)}';
```

Comment:
same question here. We probably want the same key for all table of this type 'ClassesTableSingle'

Reply:
Added TODO.

Reply:
I expect this will completely break the sorting behavior you wanted (maintaining sort direction / column between snapshots) in the current state.

Reply:
Yes, I just do not want to work on this in this PR.